CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - [Backup NG] Ability to cancel a running backup job [#3047](https://github.com/vatesfr/xen-orchestra/issues/3047)
 - [Backup NG form] Ability to enable/disable a schedule [#3062](https://github.com/vatesfr/xen-orchestra/issues/3062)
 - New backup/health view with non-existent backup snapshots table [#3090](https://github.com/vatesfr/xen-orchestra/issues/3090)
+- [Backup NG logs] Ability to report a failed job and to copy to clipboard its log (PR: [#3110](https://github.com/vatesfr/xen-orchestra/pull/3110)) [#3100](https://github.com/vatesfr/xen-orchestra/issues/3100)
```

Comment:
Please add `(PR [#3110](https://github.com/vatesfr/xen-orchestra/pull/3110/files))` please :slightly_smiling_face:

Reply:
My bad, you've already done it, maybe move it at the end though.